src/com/facebook/buck/jvm/kotlin/PathMatchers.java

```diff
@@ -22,5 +22,7 @@
 
   public static final PathOrGlobMatcher KOTLIN_PATH_MATCHER = new PathOrGlobMatcher("**.kt");
 
+  public static final PathOrGlobMatcher SRC_ZIP_MATCHER = new PathOrGlobMatcher("**.src.zip");
```

Comment:
A test `src.zip` file would be nice as a reference that this is possible. Also helps ensure it never breaks.

Reply:
Sure let me add a test